R/Seurat_III.R

```diff
@@ -1433,23 +1433,39 @@ FilterCloneNames <- function(seuratObj, minValue) {
 #' @param varName The resolution/ident to use
 #' @param genes A vector of genes to include
 #' @param slot The slot to use, passed to GetAssayData
+#' @param SerMethod This function now defaults to the base Seurat method set to T, unless when a simple average is needed set to F.
 #' @return A data.frame of avg expression per var
 #' @importFrom Matrix rowMeans
 #' @export
-AvgCellExprs <- function(seuratObj, varName = "ClusterNames_0.2", genes, slot = "scale.data"){
+AvgCellExprs <- function(seuratObj, varName = "ClusterNames_0.2", genes, slot = "data", SerMethod=T){
```

Comment:
is there a specific reason to change our default slot?  i dont really care, but want to be sure this is deliberate.  Also, can you keep casing consistent, so serMethod, not SerMethod.  I might also call it "seuratMethod", but that's me.

Reply:
the default for Seurat is counts not scaled counts.. being consistent with their method..